feature_engine/base_transformers.py

```diff
@@ -154,9 +155,18 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
 
         return X
 
-    def get_feature_names_out(self) -> List:
+    @Substitution(
```

Comment:
This is genius!  I didn't know that I could also use this class to substitute in methods! Thank you!